src/main/java/com/google/sps/data/RatingHistory.java

```diff
@@ -0,0 +1,33 @@
+package com.google.sps.data;
+
+import java.util.Date;
+import java.util.List;
+
+/** 
+ * RatingHistory Class
+ * This object class is used to track the history of a locations aggregate score each month
+ * Used as an object in the PlaceReviews
+ */
+public class RatingHistory {
+
+  private final Date start;
+  private int length;
+  private List<Double> history;
+
+  /** 
+   * Constructor
+   * Requires a start date for its constructor
+   */
+  public RatingHistory(Date start) {
+    this.start = start;
+  }
+
+  /** 
+   * Accessor Method
+   * Returns the array of aggregates
+   */
+  public List<Double> getHistory() {
+    return this.history;
+  }
+  // TODO: Implement an updater function
```

Comment:
General Tip, simple methods like getter and updater don't need to be created if they aren't in use. So the commit that requires the use of updating/getting a function can include the method's definition whenever the time comes to implement a use-case for that method. 

The idea being, methods that aren't being used aren't being created, so itll limit the size of the codebase.